script/prettier.ts

```diff
@@ -11,7 +11,12 @@ const root = Path.dirname(__dirname)
 const prettier = process.platform === 'win32' ? 'prettier.cmd' : 'prettier'
 const prettierPath = Path.join(root, 'node_modules', '.bin', prettier)
 
-const args = ['**/*.scss', '--list-different']
+const args = [
+  '**/*.scss',
+  'app/**/*.{ts,tsx}',
+  'script/**/*.ts',
+  '--list-different',
+]
```

Comment:
Let’s add `package.json` to this list — Prettier 1.13 now prints it in the same way Yarn and npm do, and it would be nice to keep `package.json` formatted correctly so that someone installing a new dependency doesn’t end up changing the file’s formatting.

Reply:
I'm gonna leave out any additive changes for follow-up PRs - this was mostly for when we jump up to TS 2.9 and start exploring some of the new syntax options.